src/OpenApi/src/Extensions/ApiDescriptionExtensions.cs

```diff
@@ -70,4 +73,34 @@ public static string MapRelativePathToItemPath(this ApiDescription apiDescriptio
         }
         return strippedRoute.ToString();
     }
+
+    /// <summary>
+    /// Retrieves the form parameters from the ApiDescription, if they exist.
+    /// </summary>
+    /// <param name="apiDescription">The ApiDescription to resolve form parameters from.</param>
+    /// <param name="formParameters">A list of <see cref="ApiParameterDescription"/> associated with the form parameters.</param>
+    /// <returns><see langword="true"/> if form parameters were found, <see langword="false"/> otherwise.</returns>
+    public static bool TryGetFormParameters(this ApiDescription apiDescription, out IEnumerable<ApiParameterDescription> formParameters)
+    {
+        formParameters = apiDescription.ParameterDescriptions.Where(parameter => parameter.Source == BindingSource.Form || parameter.Source == BindingSource.FormFile);
+        return formParameters.Any();
```

Comment:
FWIW, I think you can pass a predicate to `Any` directly.

Reply:
Yeah, I wanted to be able to resolve the parameters that matched the predicate as well. Hence the `Where` and `Any`.